src/main/webapp/app/exam/participate/exam-cover/exam-participation-cover.component.html

```diff
@@ -121,10 +121,22 @@ <h2 class="col-md-6" style="text-align: center; font-weight: normal">
             } @else {
                 @if (handInEarly) {
                     <div class="mb-2 font-weight-bold text-danger">
+                        <span jhiTranslate="artemisApp.examParticipation.handInEarly" style="font-weight: 900"></span>
+                        <br />
                         <span jhiTranslate="artemisApp.examParticipation.handInEarlyNoticeFirstSentence"></span>
                         <br />
                         <span jhiTranslate="artemisApp.examParticipation.handInEarlyNoticeSecondSentence"></span>
                     </div>
+                    @if (!isAttendanceChecked) {
+                        <div class="mb-2 font-weight-bold text-danger">
+                            <span jhiTranslate="artemisApp.examParticipation.attendanceCheck" style="font-weight: 900"></span>
+                            <br />
+                            <span jhiTranslate="artemisApp.examParticipation.attendanceNotVerifiedWarningFirstSentence"></span>
+                            <br />
+                            <span jhiTranslate="artemisApp.examParticipation.attendanceNotVerifiedWarningSecondSentence"></span>
+                            <br />
```

Comment:
One could be more specific in naming the translations as `..FirstSentence` and `...SecondSentence` is not very obvious

Reply:
this is also a convention used for hand-in early notice which can be seen above the attendance check warning